components/announcement_bar/configuration_bar/configuration_bar.tsx

```diff
@@ -213,7 +225,83 @@ const ConfigurationAnnouncementBar: React.FC<Props> = (props: Props) => {
             );
         }
 
-        if (isLicenseExpiring(props.license) && !props.dismissedExpiringLicense) {
+        if (isTrialLicense(props.license) && (isLicenseExpiringIn(props.license, 14) || isLicenseExpiringIn(props.license, 3) || isLicenseExpiringIn(props.license, 1))) {
```

Comment:
I may not be clear on requirements. 

Is this supposed to show only on days 14, 3 and 1. If so, I don't think it is implemented correctly. If I followed the logic in `isLicenseExpiringIn` correctly. It will return `true` any time expiration < 14 days not when expiration == 14 days.

If not, then you don't need to check for days 3 and 1 because checking 14 will return true. 

Would it make more sense to change this to determine the number of days to expiration. Then compare that. Rather than call `isLicenseExpiringIn` up to 4 times.

Reply:
done!